docs/src/appendix/versioning.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
is there a corresponding website PR to add this to the menu.yml?

Reply:
I was just planning to do it when opening the PR that bumps to include this.